Atomic/atomic.py

```diff
@@ -711,7 +709,27 @@ def get_col_lengths(_images):
                                               "CREATED",
                                               "VIRTUAL SIZE",
                                               "TYPE"))
+
+            for image in image_data:
+                util.write_out(col_out.format(image['indicator'], image['repo'], image['tag'], image['id'], image['created'], image['virtual_size'], image['type']))
+            util.write_out("")
+
+
+
+
+
+    def images(self):
+        _images = self.get_images()
+
+        used_image_ids = [x['ImageID'] for x in self.get_containers()]
+        all_image_info = [_images]
```

Comment:
why do we need this?  IIUIC the element in the first position is different than the rest?